### ISSUE TITLE
Hidden Icon Section just gone (not expandable) or cannot be collapsed 🤨

### ISSUE THREAD

**Issue by patrickschaper** [dwarvesf/hidden#208]
### Describe the bug

Hidden Bar is started and icons are hidden but now all hidden icons are gone and I cannot expand the hidden icons section because there is no chevron to expand on. 

![image](https://user-images.githubusercontent.com/550213/169474112-e3aabc4c-01e7-4d95-9c85-d91f3e3ce45f.png)

### Steps to reproduce

All the steps I did.

1. I installed the Mac App Store version. 
2. Moved the separation bar to the desired position.
3. Ticked "Start on login" and unticked "Show preferences on Start"
4. Because to-be-hidden section did not collapse, I removed Hidden Bar.app from my Applications folder.
5. Installed brew version and started it.

Now all icons are hidden but there is no section to expand. Clicking anywhere does not help, so it's also not invisible. It's just not there.

### Expected behaviour

I should see a chevron to expand the hidden icons section.

### Environment

Hidden Bar version 1.9
MacOS: Big Sur 11.6.6
MacBook Pro 16-inch 2019

**Comment by patrickschaper** [comment]
Some news
- After reinstalling the broken App Store app again
- I was able to set a shortcut and use it to open hidden section again
- The I reinstalled the brew version (1.9) again (after deleting the old one) 
- But now I'm having the same problem as in the App Store version... The icon cannot be hidden. 

![image](https://user-images.githubusercontent.com/550213/169481282-077f87f9-6aeb-4069-ba81-4b4552018fd9.png)

**Comment by axeloz** [comment]
I installed the AppStore version and icons never hide. I can see the chevron though. 
Monterey latest. 
Macbook Pro 15

**Comment by ShlomoCode** [comment]
Hi, this is my problem too! there is a solution?

https://user-images.githubusercontent.com/78599753/180628080-5f5bcd6a-df46-4152-86eb-5e6ea60456ea.mov

**Comment by simesy** [comment]
Yeah this happened to me and I can't seem to "reset" the app. It's continuing to work on my second macbook. If i uninstall completely, when i reinstall it remembers my settings but remains buggy. I wouldn't mind knowing how to reset the configuration data that it must be saving somewhere.

**Comment by tieubao** [comment]
Fixed in develop (commit b2ec0aa, PR #363): the arrow and separator now restore themselves at every launch, so a Cmd-dragged-off control comes back on next start instead of bricking the app. Ships in the next release; until then the manual fix is `defaults delete com.dwarvesv.minimalbar` and relaunch.
